cmd/gencosmosdb/gencosmosdb.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"unicode"
 
+	_ "github.com/go-bindata/go-bindata"
```

Comment:
This is not being used directly as a library in `go-cosmosdb`. It is just a dev tool. So it should not be imported here. I think we should use the same pattern as in ARO-RP (with `deps.go`).

Reply:
Code updated, thanks Mikalai.